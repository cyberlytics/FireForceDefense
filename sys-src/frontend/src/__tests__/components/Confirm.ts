import { mount } from '@vue/test-utils';
import Confirm from '@components/confirm.vue';

describe('Confirm.vue', () => {
    test('confirm matches snapshot', () => {
        const wrapper = mount(Confirm, {
            propsData: {
                question: 'snapshot-test-question',
                yes: 'snapshot-test-yes',
                no: 'snapshot-test-no',
            },
            mocks: {
                $t: (key: string) => key,
            },
        });
        expect(wrapper.element).toMatchSnapshot();
    });
});
