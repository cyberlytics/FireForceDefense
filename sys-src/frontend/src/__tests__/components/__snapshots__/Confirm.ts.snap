// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm.vue confirm matches snapshot 1`] = `
<div>
  <p
    class="text-center"
  >
    snapshot-test-question
  </p>
   
  <div
    class="d-flex flex-row justify-content-between"
  >
    <div>
      <button
        class="btn btn-danger btn-block btn-lg"
      >
        
                snapshot-test-yes
            
      </button>
    </div>
     
    <div>
      <button
        class="btn btn-success btn-block btn-lg"
      >
        
                snapshot-test-no
            
      </button>
    </div>
  </div>
</div>
`;
